.github/workflows/ci.yml

```
@@ -60,7 +61,7 @@ jobs:
         virtualenvs-in-project: true
 
     - name: Set up Python
-      uses: actions/setup-python@v4
+      uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5
```

Although this is the correct commit hash, what if we can use the version string instead?

```suggestion
      uses: actions/setup-python@v5.1.0
```

Reference: https://github.com/actions/setup-python/releases/tag/v5.1.0